packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1463,10 +1463,14 @@ class TextInputConnection {
 
   /// Platform sent a notification informing the connection is closed.
   ///
-  /// [TextInputConnection] should clean current client connection.
+  /// [TextInputConnection] should clean the current client connection.
```

Comment:
The connection should also report this information, and allow a listener to listen for it. It's not the connection, itself, that really needs this info, it's the object that app/package object that owns the connection that needs this info.